docs/references/expo/offline-support.mdx

```diff
@@ -71,3 +71,45 @@ To enable offline support in your Expo app, follow these steps:
   }
   ```
 </Steps>
+
+## Handling network errors
```

Comment:
There is already an [error handling](https://clerk.com/docs/custom-flows/error-handling) section. Any reason this is not included in that doc?

Reply:
Hmm, this is specific to network errors. 
Until now, in non-standard browsers we used to mute these errors and Expo users could not handle them. Using this new experimental property prevents the muting and throws these network errors, so that the developers can handle them. So, it's somewhat specific to this feature.

Reply:
Got it. I see there is a mention of this above. When I reviewed I just looked at the PR and not the whole page. Maybe this information should be added to the 'How to handle network errors' section. I can definitely see some developers missing the line in the first section.